section_faiss_vector_index.go

```diff
@@ -275,6 +275,8 @@ func (v *vectorIndexOpaque) mergeAndWriteVectorIndexes(sbs []*SegmentBase,
 	indexes []*vecIndexMeta, w *CountHashWriter, closeCh chan struct{}) error {
 
 	vecIndexes := make([]*faiss.IndexImpl, 0, len(sbs))
+	finalVecIDCap := 0
```

Comment:
nit: you can initialise both these vars in a single line as `var finalVecIDCap, indexDataCap int`